packages/stacked_cli/lib/src/services/analytics_service.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:usage/usage_io.dart';
+
+class AnalyticsService {
+  final AnalyticsIO _analytics = AnalyticsIO(
+    'UA-XXXXXXXX-X',
```

Comment:
Can we move this into a secure place? Is that required? 

If someone malicious sees this they might spam us with analytics 😅